example/w-nginx/default.conf

```diff
@@ -0,0 +1,30 @@
+server {
+  listen 8081 default_server;
+
+  sendfile off;
+  access_log  /dev/stdout;
+  error_log /dev/stderr;
+
+  root /www/data;
+
+  location /static/ {
+    # Serve static assets from the folder /www/data/static on the
+    # nginx server.
+  }
+
+  location / {
+    # Forward any other request to the dream server.
+    #
+    # Note: If you wish to provide server access behind another route,
+    # this can be accomplished by changing location above (say, to
+    # /app) and then adding a rewrite rule like this:
+    #
+    # rewrite ^/app/(.*) /$1 break;
+    #
+    # Note: In the proxy_pass command below, the "web" in
+    # http://web:8080 need to match the docker container name and port
+    # exposed in docker-compose.yml.
```

Comment:
IMO this comment should be worked into the example text somehow, where it is much more visible, and that will also be a good place to link to the `.conf` file and show snippets of it. Based on the content, you will also have good reason to link to the `docker-compose.yml` file. Let me know if you want me to merge the PR "early" so you can easily get accurate permalinks to the files for a second round of editing. I usually do examples that way myself: I write and push something approximate, then go back and get the right permalinks for everything from the browser, and hyperlink it all together.